### PR TITLE
Update to match required Tensorflow versions

### PR DESCRIPTION
**Pull request by brendenconnors**
layers.Normalization() does not exist in Tensorflow 2.3.0, which is listed as a requirement.

And Tensorflow_probability 0.11.0 (listed) is stable against tf 2.3.0 shown here https://github.com/tensorflow/probability/releases?page=2

### CODE REVIEW

**Comment by fchollet** [comment]
We should target the latest releases whenever possible. Please try it with TF 2.0 and update the requirements accordingly.

**Comment by brendenconnors** [comment]
Ran with updated versions and ran correctly without the "experimental" reference. 😃

**Review by fchollet — APPROVED** [review]

**Comment by fchollet** [comment]
Please also update the `md` and `ipynb` files.

**Comment by brendenconnors** [comment]
Arxiv link caused issue when creating md/ipynb files. Updated link in .py file and generated md/ipynb files.

**Review by fchollet — APPROVED** [review]
LGTM, thank you!

**Comment by fchollet** [comment]
There are some failing tests related to special characters:

> UnicodeDecodeError: 'utf-8' codec can't decode byte 0x95 in position 704: invalid start byte
The command '/bin/sh -c python autogen.py make' returned a non-zero code: 1
make: *** [Makefile:2: container-test] Error 1

Did you use a non-utf8 character somewhere?

**Comment by brendenconnors** [comment]
Hmm no not that I am aware of. 

Seeing this Specials character as opposed to the dot locally. 
image

For this part of the displayed page 
image

Not really sure when this would happen during generation

File: examples/generative/md/real_nvp.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 **Description:** Estimating the density distribution of the "double moon" dataset.
 
 
-<img class="k-inline-icon" src="https://colab.research.google.com/img/colab_favicon.ico"/> [**View in Colab**](https://colab.research.google.com/github/keras-team/keras-io/blob/master/examples/generativeipynb/real_nvp.ipynb)  <span class="k-dot">•</span><img class="k-inline-icon" src="https://github.com/favicon.ico"/> [**GitHub source**](https://github.com/keras-team/keras-io/blob/master/examples/generativereal_nvp.py)
+<img class="k-inline-icon" src="https://colab.research.google.com/img/colab_favicon.ico"/> [**View in Colab**](https://colab.research.google.com/github/keras-team\keras-io\blob\master\examples\generative/ipynb/real_nvp.ipynb)  <span class="k-dot">•</span><img class="k-inline-icon" src="https://github.com/favicon.ico"/> [**GitHub source**](https://github.com/keras-team\keras-io\blob\master\examples\generative/real_nvp.py)
```
Comment (fchollet) [inline examples/generative/md/real_nvp.md:9]: Links like this one should only have forward slashes. Perhaps this is an issue with the generation script on Windows (it was written to be run on Unix systems only). Please fix by hand.

**Review by brendenconnors — COMMENTED** [review]
Changed back slashes to forward slashes

File: examples/generative/ipynb/real_nvp.ipynb [hunk 2]
```diff
@@ -322,7 +323,7 @@
  "metadata": {
   "colab": {
    "collapsed_sections": [],
-   "name": "real_nvp",
+   "name": "..\\examples\\generative\\real_nvp",
```
Comment (fchollet) [inline examples/generative/ipynb/real_nvp.ipynb:326]: Please revert this line

File: examples/generative/md/real_nvp.md [hunk 3]
```diff
@@ -850,9 +859,13 @@ axes[1, 1].set_ylim([-2, 2])
 
 ```
 </div>
-![png](/img/examples/generative/real_nvp/real_nvp_13_1.png)
+    
+![png](../examples/generative/md/real_nvp_files/../examples/generative/md/real_nvp_13_2.png)
```
Comment (fchollet) [inline examples/generative/md/real_nvp.md:863]: The image paths are incorrect -- they should be reverted to the previous format.

**Comment by brendenconnors** [comment]
updated changes to image path and metadata name. 👍

**Review by fchollet — APPROVED** [review]
Thank you!

**Pull request merged**